lib/diagnostic.rb

```diff
@@ -28,18 +33,24 @@ def example
 # responds successfully and lists all examples.
 
 RSpec.describe 'Examples API' do
-  # your test(s) here
+    it 'gets /examples' do
+      get '/examples'
+
+      expect(response).to be_success
+    end
 end
 
+
 #
 # Question 3
 #
 # Create a routing spec for our `Examples` resource that ensures
 # GET /examples/:id routes to the examples#show action.
 
 RSpec.describe 'routes for examples' do
-  # your test(s) here
-end
+  it 'routes GET /examples to the examples#show action' do
+    expect(get('/examples')).to route_to('examples#show')
+  end
```

Comment:
When we have a route to test with route params, we need to route_to our controller actions with something like the following:

```
  it 'routes GET /examples/:id to the examples#show action' do
    expect(get('/examples/1')).to route_to(
      controller: 'examples',
      action: 'show',
      id: '1'
    )
  end
```